nncf/ptq/examples/mobilenet_v2/usercode.py

```diff
@@ -0,0 +1,100 @@
+# This code was taken as is from the pytorch repository. You
```

Comment:
let's call this file `utils.py` because, frankly, this is not the only code the user needs to write.

Reply:
Done.